config/brew/typedb.rb

```diff
@@ -15,7 +15,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-class TypeDB < Formula
+class Typedb < Formula
```

Comment:
With formula name that doesn't match Homebrew convention, users would not be able to install TypeDB and installation would fail with this error:

```
In formula file: /usr/local/Homebrew/Library/Taps/vaticle/homebrew-tap/Formula/typedb.rb
Expected to find class Typedb, but only found: TypeDB.
```

Therefore, we're renaming the formula to what Homebrew expects.